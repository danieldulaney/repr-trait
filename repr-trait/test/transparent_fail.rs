use repr_trait::Transparent;

#[derive(Transparent)]
struct NotTransparent {
}

fn main() {}
