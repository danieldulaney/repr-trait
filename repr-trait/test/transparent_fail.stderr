error: proc-macro derive panicked
 --> $DIR/transparent_fail.rs:3:10
  |
3 | #[derive(Transparent)]
  |          ^^^^^^^^^^^
  |
  = help: message: Can't derive Transparent on a struct without #[repr(transparent)]
